app/components/standup_meeting/meeting_update_component.html.erb

```diff
@@ -1,5 +1,8 @@
-<div class="border p-4 flex flex-col lg:flex-row justify-between mt-4">
-  <span class="flex flex-col justify-center mr-4 font-bold"><%= user.email %></span>
+<div class="border p-4 flex flex-col  justify-between mt-4">
+  <span class="flex flex-col justify-center mr-4 font-bold">
+    <%= user.full_name %>
+  </span>
+  <hr class="border-t border-gray-400 my-4" />
```

Comment:
nit, but so you should know -- you can use the native daisy class:
`<div class="divider my-4 "></div>`

Reply:
done